docs/tracing_ui.md

```diff
@@ -0,0 +1,210 @@
+| ----------------------------| -------------                         |
+| 1                           | Toggle Visibility of all Trees        |
+| 2                           | Toggle Visibility of Inactive Trees   |
+| SHIFT + ALT + Left Click    | Merge Two Nodes and Combine Trees     |
+| SHIFT + CTRL + Left Click   | Delete Edge / Split Trees             |
+
+![Trees can be hidden for a better overview over the data. Toggle the visibility of individual tree using the checkbox in front of the tree's name or use the button to toggle all (inactive) trees at once.](images/tracing_ui_tree_visibility.png)
+
+
+### Importing & Exporting NML Files
+webKnossos makes it easy to import or export skeleton tracings as [NML files](./data_formats.md#nml). There are two ways for downloading your tracings:
+
+1. There is a `Download` button in the overflow menu next to the prominent `Save` button in the toolbar at the top of the screen. 
+
+2. If you need more fine-grained control over which trees to download use the `Download Selected Trees` option. From the `Trees Tab` click on `More` and select `Download Selected Trees` from the menu. All visible trees (checkmark in front of the name) will be downloaded as an NML file. This is especially useful if you need to only download a single tree of an otherwise much larger tracing.
+
+![Skeletons can be exported and downloaded as NML files from the tracing view. Either download all or only selected trees. ](images/tracing_ui_download.png)
+
+Importing a skeleton tracing can be achieved using two approaches as well:
+
+1. If you already have a tracing open you can easily add more skeletons to it by drag and dropping an NML file onto your browser window. Otherwise, use the `Import NML` option next to `Download Selected Trees`. This will merge the NML file's content with the already open tracing. 
+
+2. To import a skeleton tracing as a completely new wK annotation, just drag and drop the NML file anywhere on your user dashboard. Alternately, navigate to your user dashboard and use the `Upload Annotation` button within the "Explorative Tracings" section. 
+
+![Skeletons can be imported by drag and drop in the tracing view or from the dashboard](images/tracing_ui_import.png)
+
+### Merging Skeleton Annotation
+There are two ways for merging tracings:
+
+1. While in the tracing UI, drag and drop an NML file onto your browser window to import a skeleton. The imported skeleton will be merged with currently open tracing.
+
+2. If you would like to merge your current tracing with another existing tracing, select the `Merge` operation from the overflow menu next to the `Save` button. (see image) Either enter the ID of an existing explorative tracing or select a whole project and proceed to merge the selection with your currently open tracing. The resulting annotation can either be created as a new explorative tracing. Otherwise, the merge will happen in your current tracing. 
```

Comment:
I'd prefer "either ... or ..." in the same sentence. Having an additional sentence for the second part (leading with "Otherwise") sounds weird to me, but I'm not a native, so....